debian/patches/fix-arch-on-32-bit.patch

```diff
@@ -0,0 +1,49 @@
+From 4611b33f9393214a39295cdde3c1ec5ad410b10d Mon Sep 17 00:00:00 2001
```

Comment:
This patch should be applied in the main branch, instead of going through the debian packaging indirection.